app/Http/Controllers/RobotsController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class RobotsController extends FrontController
+{
+    public function index(Request $request)
+    {
+        if (config('app.env') === 'production' && config('app.url') === $request->getHttpHost()) {
```

Comment:
Laravel provides a way to check the app's environment:
```php
if (app()->environment('production') && ... ) {
```
https://laravel.com/docs/8.x/configuration#determining-the-current-environment

They're both equivalent, but I think the Laravel way reads a little nicer.

Reply:
I thought the same and initially had that, but in unit tests, `app()->environment('production')` will always be testing, so I was finding it hard to unit test production behavior. If you have thoughts on how to override, let me know! I'll merge for now.